client/overview/task-list/index.js

```diff
@@ -32,20 +175,37 @@ const TaskList = ( { tasks } ) => {
 			</CardHeader>
 			<CardBody>
 				<CollapsibleList
+					animation="slide-right"
```

Comment:
Soo unfortunately this doesn't work, I noticed a couple bugs in the collapsible list and task item just in relation to animations (so it shouldn't be a blocker). The animation prop doesn't do anything at the moment, as the `task-item` doesn't pass this property down.
You could change it back to `custom` and add this css -> https://github.com/woocommerce/woocommerce-admin/blob/main/client/task-list/task-list.scss.
Now there is also a caveat with that unfortunately, it appears that animations are broken for any task list items that would be collapsed, but the animation works for the items that are always displayed 😕 (although I found a fix)

So we might want to leave it as `custom` and add the above css, then things should work perfectly after the remaining bugs are fixed within the experimental package.

Reply:
Hi @lowie17,

This is interesting. If I change the prop to `custom`, the animation starts working. I guess that it's using [this css](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/experimental/src/experimental-list/style.scss#L33-L60).

On the other hand, the animation looks odd when I add [the css](https://github.com/woocommerce/woocommerce-admin/blob/main/client/task-list/task-list.scss) we have in WC-Admin task list

![Screen Capture on 2021-07-13 at 10-14-28](https://user-images.githubusercontent.com/1314156/125460428-a085e726-4834-4e0a-871d-ba3d43899710.gif)

Another thing I realized is that there is a missing border in some list items. This is happening on `Payments` > `Overview`:

![screenshot-localhost_8082-2021 07 13-10_28_45](https://user-images.githubusercontent.com/1314156/125460581-120bb96c-9492-4828-a433-855043d196be.png)

But also it's happening on `WooCommerce` > `Home`
![screenshot-one wordpress test-2021 07 13-10_36_27](https://user-images.githubusercontent.com/1314156/125461814-b5f5788f-ee33-483e-b2dd-7fadc01c4f3b.png)

I think that the issue is related to [this line](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/experimental/src/experimental-list/style.scss#L110).

Reply:
> This is interesting. If I change the prop to custom, the animation starts working. I guess that it's using this css.
> On the other hand, the animation looks odd when I add the css we have in WC-Admin task list

Interesting indeed, `custom` should keep the transitions disabled. For me locally the transitions where still disabled with your latest changes. And adding [this css](https://github.com/woocommerce/woocommerce-admin/blob/main/client/task-list/task-list.scss) seemed to work nicely:

![wc-pay-task-list-animations](https://user-images.githubusercontent.com/2240960/125504174-9afda465-782c-4ef2-a18d-cca22676e374.gif)

I am curious if there is a conflict with the css loading, that we are seeing different things? Maybe we want to allow for a class name to be overwritten so there is no css conflicts?

Nice catch on the border issue, I guess that might be related to the container of the collapsed items?